.github/actions/scripts/save-coredump.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+bucket_name=$S3_BUCKET_NAME
+bucket_prefix=$S3_BUCKET_TEST_PREFIX
```

Comment:
it'll be possible to find that from the test logs, right?

Reply:
yes, you can see the prefix and file name from the logs.